breeze/models/user.py

```diff
@@ -1,11 +1,32 @@
 class User:
+    """The basic User class
+    """
     def __init__(self, username, password, role):
         self.__username = username
```

Comment:
On the brief, we don't have a username attribute but we need real names. Maybe split the username into first_name and last_name and use email instead of username for login?

Reply:
Ah, sorry for missing that in the brief..! I just checked the coursework guidelines, and it says, '_All user types should be able to log in and log out using a username and password pair_.' I'm not sure if this is strictly required, or if we could consider alternative methods. What do you think? Using email for login could be a good solution if allowed!